2001-06-22  Jan Hubicka  <jh@suse.cz>

	* i386.c (x86_use_loop): Disable for K6 temporarily.

--- gcc/config/i386/i386.c.jj	Wed Jun 13 16:38:39 2001
+++ gcc/config/i386/i386.c	Fri Jun 22 19:41:41 2001
@@ -205,7 +205,9 @@ const int x86_cmove = m_PPRO | m_ATHLON;
 const int x86_deep_branch = m_PPRO | m_K6 | m_ATHLON;
 const int x86_use_sahf = m_PPRO | m_K6 | m_ATHLON;
 const int x86_partial_reg_stall = m_PPRO;
-const int x86_use_loop = m_K6;
+/* Should be set to K6 and i386, but is broken
+   and temporarily disabled for 2.96-RH and 3.0.x.  */
+const int x86_use_loop = 0; 
 const int x86_use_fiop = ~(m_PPRO | m_ATHLON | m_PENT);
 const int x86_use_mov0 = m_K6;
 const int x86_use_cltd = ~(m_PENT | m_K6);
